docs: add 1.11 changelog (#1496)
software-mansion/radon-ide: packages/docs/docs/_changelog/1.11.x.md

---
version: "1.11.x"
date: "2025-08-27"
title: "Device Files Transfer, Improved Network Inspector, Expo SDK 54"
---

- **Device File Transfer**: You can now send files to device.

<video autoPlay loop width="700" controls className="shadow-image changelog-item">
  <source src="/video/ide_send_files.mp4" type="video/mp4" />
</video>

> - **Improved Network Inspector**: Now supporting extended filtering, sorting alongside improvements to UX. Also the list of requests is now more performant when dealing with a large number of requests.

Reviewer: ```suggestion
- **Improved Network Inspector**: Added support for extended request filtering and sorting alongside various other UX improvements. Improved performance when dealing with a large number of requests.
```

<img
  width="600"
  src="/img/docs/ide_network_inspector_filter.png"
  className="shadow-image changelog-item"
/>

- **Expo SDK 54 support**

> - Added `stepInto`/`stepOut` controls to debugger

Reviewer: ```suggestion
- Added "Step Into"/"Step Out" controls to the debugger overlay
```

> - You can now configure Expo Prebuild to run before building the application and control Replay using keyboard.

Reviewer: ```suggestion
- Expo Prebuild can now be configured to run before building the application

- Replay overlay can now be controlled with the keyboard
```
